devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -248,7 +253,10 @@ private void createGradleWrapper(File projectDirectory, Properties props) {
                 getLog().warn("Unable to install the Gradle wrapper (./gradlew) in project. See log for details.");
             }
 
-        } catch (InterruptedException | IOException e) {
+        } catch (IOException e) {
+            getLog().warn("Unable to install the Gradle wrapper with gradle. Gradle wrapper is going to be copy.");
+            copyGradleWrapperFiles(platform, writer);
```

Comment:
Shouldn't we just drop the part where we try to use a local Gradle? I think copying things is better.